### ISSUE TITLE
ESC/POS example broken 2.1 branch

### ISSUE THREAD

**Issue by lite1979** [qzind/tray#462]
Logs are blank when attempting to send ESC/POS commands to an Epson TM-T88V. 

From Firefox browser console:

```
TypeError: data[i].type is undefined
qz-tray.js:574:42
    relative file:///C:/Program Files/QZ Tray/demo/js/qz-tray.js:574
    print file:///C:/Program Files/QZ Tray/demo/js/qz-tray.js:1168
    printCommand file:///C:/Program Files/QZ Tray/demo/sample.html:1326
    onclick file:///C:/Program Files/QZ Tray/demo/sample.html:1

```
​

**Comment by tresf** [comment]
I'm able to reproduce.  It looks like it's getting hung up in the backwards compat logic when the first element of the data array is an object.  @bberenz can you take a look?  It doesn't seem to happen for the other raw examples.

**Comment by tresf** [comment]
Closed via 62d70e7
